bugwarrior/config.py

```diff
@@ -11,6 +11,11 @@
 log = logging.getLogger(__name__)
 
 
+# The name of the environment variable that can be used to ovewrite the path
+# to the bugwarriorrc file
+BUGWARRIORRC = "BUGWARRIORRC"
```

Comment:
This variable doesn't seem necessary to me (not likely to be mistyped or changed) but your call.